docs/source/package_reference/hf_api.mdx

```diff
@@ -2,46 +2,53 @@
 
 [[autodoc]] login
 
-[[autodoc]] HfApi.logout
+[[autodoc]] logout
```

Comment:
Are we suggesting people to use these methods instead of `HfApi`? I would think exposing `HfApi` as the primary point for users would make more sense though. Especially if we go with not storing the token automatically and storing it in the instance.

Reply:
I've added only `HfApi` for now, with an introduction of how each method can still be imported from the root - this was the preferred way up to now as simpler, but it might not be if we start storing the token automatically, so I've added a mention of both. Let me know if that works for you.